src/modules/settings/SecurityPage.tsx

```diff
@@ -0,0 +1,38 @@
+import { Box, Container } from '@mui/material';
+import { HalfTypeNavigation } from '../../shared-components/Navigation/TopNavigation/HalfTypeNavigation';
+import { colors } from 'src/styles/colors';
+import { ListItemSelect } from '../../shared-components/Card/ListItems/ListItemSelect';
+import { useState } from 'react';
+import { Security } from 'src/types/Security';
+import { UserActionType, useUserDispatch } from 'src/context/UserContext';
+
+export const SecurityPage = () => {
+  const [selectedSecurity, setSelectedSecurity] = useState<Security | null>(
```

Comment:
Same here, use data from context instead of using local state.

Reply:
Fixed